src/BaseClient.php

```diff
@@ -423,6 +423,10 @@ private function prepareAndExecuteRequest(string $method, string $url, array $op
             'Authorization' => sprintf('Bearer %s', $this->accessToken->getToken()),
         ];
 
+        if (! empty($options['language'])) {
```

Comment:
- for code standard remove the space after `!` 
- specify a comment: `// pass through Accept-Language header`

Reply:
Updated, thx!